src/VirtoCommerce.Platform.Web/Controllers/Api/SecurityController.cs

```diff
@@ -532,6 +532,18 @@ public async Task<ActionResult<SecurityResult>> Update([FromBody] ApplicationUse
             {
                 return Ok(IdentityResult.Failed(new IdentityError { Description = "It is forbidden to edit this user." }).ToSecurityResult());
             }
+
+            var applicationUser = await _userManager.FindByIdAsync(user.Id);
+            if (applicationUser.EmailConfirmed != user.EmailConfirmed)
+            {
+                var hasPermission = Request.HttpContext.User.HasGlobalPermission(PlatformConstants.Security.Permissions.SecurityVerifyEmail);
+
+                if (!hasPermission)
```

Comment:
merge with the upper if maybe?

Reply:
I do not want to execute HasGlobalPermission if emailConfirmed not changed.